source/reviews/speed.txt

```diff
@@ -0,0 +1,149 @@
+================
+Speed of Reviews
+================
+
+Why Reviews Should Be Fast
+--------------------------
+
+**At MongoDB, we optimize for the speed at which a team of authors can produce
+content together**, as opposed to optimizing for the speed at which an
+individual author can write content. The speed of an individual is
+important, it's just not *as* important as the velocity of the entire team.
+
+When code reviews are slow, several things happen:
```

Comment:
I think it's worth mentioning a number of factors that cause reviews to be slow since the reviewee can often mitigate the risk.
- Size of PR
- Reaching conclusive/acceptable solutions to PR comments quickly
- Adhering to the scope of the ticket
- Describing the changes made and any scope creep in the ticket or PR
- Proofreading and preparing PRs according to PR template guidelines